bin/mail/2016-12-12-hoc-post-survey/generate-organizers

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# All HocSignup2016 entries,
+# split into 2 location groups: US and international (all other).
+query = 'kind_s: HocSignup2016'
+
+US_VALUES = {
+  location_country_s: 'United States',
+  location_country_code_s: 'US',
+  hoc_country_s: 'us',
+  create_ip_country_s: 'United States',
+
+  # This is a free-form text field with many variations. Comparisons will be case-insensitive,
+  # so only lower case versions are needed here.
+  country_s: [
+    'united states',
+    'usa',
+    'america',
+    'united states of america',
+    'united state of america',
+    'us',
+    'u.s.a',
+    'u.s.',
+    'unitedstates',
+    'united state',
+    'united sates',
+    'united states of amarica',
+    'unites states',
+    'united stated',
+    'united stats',
+    'united  states',
+    'united staes',
+    'unided states',
+    'untited states',
+    'unite states',
+    'united staes of america',
+    'unitedstatesofamerica',
+    'unitied states',
+    'united sates of america',
+    'united stares',
+    'untied states of america',
+    'united states of amrica',
+    'united stats of amarica',
+    'unided states of america',
+    'United Atates',
+  ]
+}.freeze
+
+def us?(row)
+  US_VALUES.each do |key, values|
+    row_value = row[key.to_s]
+    next unless row_value
+
+    values = *values if values.is_a? String
+    values.each do |value|
+      return true if value.casecmp(row_value).zero?
+    end
+  end
```

Comment:
This would work. Is it more readable?

```ruby
def us?(row)
  US_VALUES.each do |key, values|
    return true if values.any? {|v| v.casecmp(row[key.to_s].to_s).zero?}
  end
  false
end
```